{
  try {
    Object bean=result;
    String beanName=(String)args[1];
    processBean(beanName,bean);
  }
 catch (  Throwable t) {
    logger.warn("Unexpected exception",t);
  }
}
